reinforcement_learning/rlclientlib/logger/async_batcher.h

```diff
@@ -37,8 +36,8 @@ namespace reinforcement_learning {
                   error_callback_fn* perror_cb = nullptr,
                   size_t send_high_water_mark = (1024 * 1024 * 4),
                   size_t batch_timeout_ms = 1000,
-                  size_t queue_max_size = (8 * 1024));
-
+                  size_t queue_max_size = (8 * 1024),
+                  bool queue_dropping_disable = false);
```

Comment:
double negation "disable + false" sounds a bit complicated. Maybe it is better either to rename _queue_dropping_disable to something like _queue_blocking_available or switch to enum with explicit mode names (queue_mode::BLOCKING / queue_mode::DROP)

Reply:
I like your idea using enum. Looks definitely better indeed.